src/ajax/check_hgvs_dialogue.php

```diff
@@ -43,15 +43,12 @@
 
 
 
-// Initialise the function that will allow us to reset all values.
-function reset_all_values()
-{
-    // This function can be called to reset all values from the
-    //  output of this script.
-    print('
-    // Resetting the md5 key.
-    $(\'input[name="codedVariants"]\').val("");
-    
+// Reset all values.
+print('
+// Resetting the md5 key.
+$(\'input[name="codedVariants"]\').val("");
+
+if ($(\'#variantForm input[name*="VariantOn"]\').hasClass("acc")) {
```

Comment:
This is THE OPPOSITE of what was said in the commit message: so the code here shows that we **are only resetting all values if there ARE values of the class `acc`epted**. The commit message implies that we would only reset "IF there were no variants successfully mapped". The issue here lies within the commit message (and me writing it wrongly). So, to elaborate: we **only want to reset if any values were set to `acc`epted**, since these values were mapped by VariantValidator and these are the values which need reconsideration after a value has been changed. We do NOT need to reconsider variants which were never changed by us in the first place; these are being checked thoroughly after the submit, so we don't need to worry about them.

My apologies for the confusing commit message here :).